app/tests/conftest.py

```diff
@@ -63,17 +66,13 @@ def payment_gateway_contract():
     )
     web3.eth.waitForTransactionReceipt(tx_hash)
 
-    web3.eth.defaultAccount = agent['account_address']
-    web3.personal.unlockAccount(agent['account_address'], agent['password'])
```

Comment:
This is not necessary when Ganache is assumed to be used as a unit test execution environment. Therefore, we have removed all of them this time.